docs/docs/what-you-dont-need-plugins-for.md

```diff
@@ -7,7 +7,7 @@ Most third-party functionality you want to add to your website will follow stand
 Some examples:
 
 - Importing JavaScript packages that provide general functionality, such as `lodash` or `axios`
-- Using React components or component libraries you want to include in your UI, such as `Ant Design`, `Material UI`, or the typeahead from your component library.
+- Using React components or component libraries you want to include in your UI or the typeahead from your component library. ⚠️ You need a plugin for [Material-UI](https://github.com/hupe1980/gatsby-plugin-material-ui) or [Ant Design](https://github.com/bskimball/gatsby-plugin-antd).
```

Comment:
Ok, if we need plugins for those libraries, then whole bullet point about packages that don't need dedicated gatsby plugin loses a lot of sense ;)

Reply:
Yes, I believe the documentation for plugins should bring clarity on situation where users need them. Because, getting a dependency from `npm` is always required. It is not clear how plugins help. A good idea might be to add a diagram to visualize _why of plugins_ in `Gatsby`.

Reply:
Should I remove the bullet point?

Reply:
> Yes, I believe the documentation for plugins should bring clarity on situation where users need them. Because, getting a dependency from `npm` is always required. It is not clear how plugins help. A good idea might be to add a diagram to visualize _why of plugins_ in `Gatsby`.

Totally support this . It's quite a struggle to know as of now that the plugins are mandatory.

Reply:
Sorry for the delay in responding to this @oliviertassinari! I'm in favor of removing the bullet, but think it might make sense to clarify the nuance with something like this: 

```suggestion
- Using React components or component libraries you want to include in your UI or the typeahead from your component library. ⚠️ In some cases with these libraries you may need a plugin to handle more advanced use cases like server-side rendering and adding style providers with [Material-UI](https://github.com/hupe1980/gatsby-plugin-material-ui), or to configure Babel for you like with [Ant Design](https://github.com/bskimball/gatsby-plugin-antd).
```

This might be a compromise to show that I could import some really simple generic component I've made and published myself, or include one of these more impressive libraries that needs to cover more edge cases and provide more customization.

Maybe @pieh has more thoughts too? It does seem a little silly to name a guide "what you don't need plugins for" and then need a plugin for one of the examples 😆

Reply:
👍 for removing the bullet.